docs/source/common/lightning_cli.rst

```diff
@@ -91,8 +91,22 @@ practice to create a configuration file and provide this to the tool. A way to d
 
 The instantiation of the :class:`~pytorch_lightning.utilities.cli.LightningCLI` class takes care of parsing command line
 and config file options, instantiating the classes, setting up a callback to save the config in the log directory and
-finally running :func:`trainer.fit`. The resulting object :code:`cli` can be used for instance to get the result of fit,
-i.e., :code:`cli.fit_result`.
+finally running the trainer. The resulting object :code:`cli` can be used for instance to get the instance of the
+model, i.e., :code:`cli.model`.
+
+The :class:`~pytorch_lightning.utilities.cli.LightningCLI` is configured to run
+:meth:`~pytorch_lightning.trainer.Trainer.fit` by default. This can be changed either by setting
+``LightningCLI(trainer_fn="test")`` or by passing the argument through command line positionally, e.g.:
+
+.. code-block:: bash
+
+    python trainer.py test --trainer.limit_test_batches=10
```

Comment:
If the trainer arguments are added the the main parser, then the subcommand would need to go after.

```suggestion
    python trainer.py --trainer.limit_test_batches=10 test
```